### PR TITLE
added missing implementation "getArgAsRgbaColor"

### PR DESCRIPTION
**Pull request by 2bbb**

### CODE REVIEW

**Comment by ofTheo** [comment]
hmm - not sure why this wasn't implemented. 
I think these lines might need to be uncommented also? 

https://github.com/openframeworks/openFrameworks/blob/master/addons/ofxOsc/src/ofxOscMessage.cpp#L470

If you want to add that to your PR I'll merge it in!
Thanks!!

**Comment by 2bbb** [comment]
@ofTheo 
oh... i forgot it!!
i pushed uncommented commit.

thanks!!

File: addons/ofxOsc/src/ofxOscMessage.cpp [hunk 1]
```diff
@@ -336,7 +336,16 @@ ofBuffer ofxOscMessage::getArgAsBlob( int index ) const
         return ((ofxOscArgBlob*)args[index])->get();
 }
 
-
+int32_t ofxOscMessage::getArgAsRgbaColor( int index ) const
+{
+    if ( getArgType(index) != OFXOSC_TYPE_BLOB )
```
Comment (ofTheo) [inline addons/ofxOsc/src/ofxOscMessage.cpp]: shouldn't this be: OFXOSC_TYPE_RGBA_COLOR ?

**Comment by 2bbb** [comment]
@ofTheo 
yeah, exactly.
i did copy & paste, and overlooked.

**Comment by ofTheo** [comment]
going to wait for the Travis CI and AppVeyor to confirm it builds. 
it would be great though if you could test that it works with an example. 

Always good when submitting a PR :)

**Comment by 2bbb** [comment]
@ofTheo 

I think to try using the travis!
thanks!!

**Comment by 2bbb** [comment]
@ofTheo 

I think to try using the travis or other Test tools!
thanks!!

**Pull request merged**